wrappers/swift/LogConfiguration.swift

```diff
@@ -0,0 +1,214 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Class wrapping `ODWLogConfiguration` ObjC class object, representing configuration related to events.
+public class LogConfiguration {
+    private var odwLogConfiguration: ODWLogConfiguration
+
+    /**
+    Constructs `self` with provided config as `ODWLogConfiguration` object.
+
+    - Parameters:
+        - config: `ODWLogConfiguration` object which would be wrapped to.
+    */
+    public init(config: ODWLogConfiguration) {
```

Comment:
Do we envisage clients using this? If not, we should make this private.

Reply:
It might get used with other swift wrappers hence keeping it public for now.

Reply:
You mean used by other classes in the same package or outside of the package? If inside the package, `internal` should be sufficient.